main.tf

```diff
@@ -191,6 +191,8 @@ module "web_app" {
   location                  = var.config_webapp_location
   webapp_repo_name          = var.config_webapp_repo_name
   webapp_release            = var.config_webapp_release
+  webapp_image_version      = var.config_webapp_image_version
+  webapp_deployment_context_env = var.config_webapp_deployment_context
```

Comment:
the format of the file seems to be a bit off run a `terraform fmt` to correct it